assets/css/components/forms/select.css

```diff
@@ -57,9 +59,15 @@ select::-ms-expand {
   clip-path: polygon(100% 0%, 0 0%, 50% 100%);
 }
 
-.select-menu:focus-within {
-  outline: 0.15em solid var(--color-accent);
-  box-shadow: var(--focus-style-box-shadow);
+@supports (selector(:focus-visible)) and (selector(:has(a))) {
+  select:focus {
+    outline: transparent;
+  }
+
+  .select-menu:has(:focus-visible) {
+    outline: 0.15em solid var(--color-accent);
+    box-shadow: var(--focus-style-box-shadow);
+  }
```

Comment:
could maybe use the button focus style here since the select button resembles a regular button somewhat.